client/my-sites/domains/components/form/country-select.jsx

```diff
@@ -16,22 +16,19 @@ import FormInputValidation from 'components/forms/form-input-validation';
 import scrollIntoViewport from 'lib/scroll-into-viewport';
 import FormSelect from 'components/forms/form-select';
 
-export default React.createClass( {
-	displayName: 'CountrySelect',
+class CountrySelect extends Component {
 
-	mixins: [ observe( 'countriesList' ) ],
-
-	recordCountrySelectClick() {
+	recordCountrySelectClick = () => {
 		if ( this.props.eventFormName ) {
 			analytics.ga.recordEvent( 'Upgrades', `Clicked ${ this.props.eventFormName } Country Select` );
 		}
-	},
+	};
 
 	focus() {
```

Comment:
A comment might be nice here, to explain why we don't need `this` binding here. :)